docs/advanced-usage/compute-backends/htcondor/index.md

```diff
@@ -1,3 +1,74 @@
 # HTCondor
 
-Use via decorators.
+HTCondor is a specialized workload management system for compute-intensive jobs
+and it is supported by REANA alongside primary job execution backend Kubernetes
+and Slurm.
+
+## Authentication
+
+In order to use CERN HTCondor cluster you need to be authenticated using
+Kerberos.
+
+### Keytab generation
```

Comment:
The section on keytab generation and secrets uploading should better be located in the Access control -> Kerberos part of the documentation.

HTCondor and Slurm can then refer to it.

(And people using Kubernetes with Kerberos can refer to it as well, rather than having to refer to HTCondor...)